tests/system/CMakeLists.txt

```diff
@@ -441,6 +451,17 @@ ENDIF()
                     0 # VMC
                     TRUE)
 
+IF(ENABLE_SOA)
```

Comment:
Why is the hybrid rep test dependent on SOA? Are there hybrid fixes not in the non-SOA code path?